### PR TITLE
RangeValidation

### PR DESCRIPTION
**Pull request by perezlg1**

### CODE REVIEW

File: front/src/common/validators/range-number.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import { FieldValidationResult } from 'lc-form-validation';
+import { any, number } from 'prop-types';
+
+export const VALIDATION_TYPE = 'FIELDS_MATCH';
```
Comment (brauliodiez) [inline front/src/common/validators/range-number.ts]: Copy paste... malou malou malou

RANGE_NUMBER

File: front/src/common/validators/range-number.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import { FieldValidationResult } from 'lc-form-validation';
+import { any, number } from 'prop-types';
```
Comment (brauliodiez) [inline front/src/common/validators/range-number.ts]: prop-types is from es6?

File: front/src/common/validators/range-number.ts [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+export const VALIDATION_TYPE = 'FIELDS_MATCH';
+
+interface CustomParams {
+    minimum: number;
+    maximum: number;
+}
+
+//export const customParams = {minimum:3, maximum: 5};
```
Comment (brauliodiez) [inline front/src/common/validators/range-number.ts]: Remove lechuguita

File: front/src/common/validators/range-number.ts [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+export const validateRangeNumber = (value: number, vm: any, customParams: CustomParams): FieldValidationResult => {
+    validateCustomParams(customParams);
+
+    const isValueInformed = (value != void (0) && value !== null);
+    let passed = true;
+
+
+
```
Comment (brauliodiez) [inline front/src/common/validators/range-number.ts]: white space pool?

File: front/src/common/validators/range-number.ts [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+const buildValidationResult = (successfull: boolean): FieldValidationResult => {
+    const fieldValidationResult: FieldValidationResult = new FieldValidationResult();
+    fieldValidationResult.type = VALIDATION_TYPE;
+    fieldValidationResult.succeeded = successfull;
+    fieldValidationResult.errorMessage = '';
+    if (!successfull) {
+        fieldValidationResult.errorMessage = 'Error. It has to be an integer between 3 and 5';
+    }
```
Comment (brauliodiez) [inline front/src/common/validators/range-number.ts]:
```typescript
fieldValidationResult.errorMessage = (successfull) ?
                                                                    'Error. It has to be an integer between 3 and 5'
                                                             : '';
```

File: front/src/common/validators/range-number.ts (hunk 1 above)
Comment (perezlg1) [reply front/src/common/validators/range-number.ts]: Done

File: front/src/common/validators/range-number.ts (hunk 3 above)
Comment (perezlg1) [reply front/src/common/validators/range-number.ts]: Done

File: front/src/common/validators/range-number.ts (hunk 4 above)
Comment (perezlg1) [reply front/src/common/validators/range-number.ts]: Done

File: front/src/common/validators/range-number.ts (hunk 2 above)
Comment (perezlg1) [reply front/src/common/validators/range-number.ts]: Done

**Pull request merged**